### PR TITLE
Work and About myself section

### PR DESCRIPTION
**Pull request by Therealadetunji**
1. I did the work section containing the work/project I have done.
2. i created the about myself section and listed things about myself and things i can do.

### CODE REVIEW

File: Portfolio.html [hunk 1]
```diff
@@ -44,6 +44,219 @@ <h3 class="thirdtext">LET’S CONNECT</h3>
...
+      <!-- WORK SECTION-->
+      <section id="card-works">
+        <div class="singlecard">
+          <img
+            class="portfolioimage"
+            src="./images/Snapshoot-Portfolio.png"
+            alt="Snapshoot-Portfolio"
+          />
+          <div>
+            <h1 class="boldtext">Tonic</h1>
```
Comment (elmar8287) [inline Portfolio.html]: - [ ] Please, consider using one `h1` tag in your code. You have it already on line #22, so all other headlines should be `hx` tag, aka h2, h3, ...

File: Portfolio.html [hunk 2]
```diff
@@ -44,6 +44,219 @@ <h3 class="thirdtext">LET’S CONNECT</h3>
...
+        </div>
+        <!--END OF FIRST CARD-->
+        <div class="singlecard">
+          <img
+            class="portfoliopic"
+            src="./images/Snapshoot-Portfolio2.png"
+            alt="2nd-Snapshoot-Portfolio2"
+          />
+          <div>
+            <h1 class="second-boldtext">Multi-Post Stories</h1>
```
Comment (elmar8287) [inline Portfolio.html]: - [ ] ... please, consider replacing `h1` to `hx` tag accordantly.

File: Portfolio.html [hunk 3]
```diff
@@ -44,6 +44,219 @@ <h3 class="thirdtext">LET’S CONNECT</h3>
...
+        </div>
+        <!--END OF second CARD-->
+        <div class="singlecard">
+          <img
+            class="portfoliopic"
+            src="./images/Snapshoot-Portfolio3.png"
+            alt="3rd-Snapshoot-Portfolio2"
+          />
+          <div>
+            <h1 class="boldtext">Tonic</h1>
```
Comment (elmar8287) [inline Portfolio.html]: - [ ] ... please, consider replacing `h1` to `hx` tag accordantly.

File: Portfolio.html [hunk 4]
```diff
@@ -44,6 +44,219 @@ <h3 class="thirdtext">LET’S CONNECT</h3>
...
+          </div>
+
+          <button class="lasttxt2">get my resume</button>
+        </div>
+
+        <div class="allinfo">
+          <div>
+            <ul class="secondlists">
+              <li>
+                <h1 class="langlist">Languages</h1>
```
Comment (elmar8287) [inline Portfolio.html]: - [ ] ... please, consider replacing `h1` to `hx` tag accordantly.

File: Portfolio.html [hunk 5]
```diff
@@ -44,6 +44,219 @@ <h3 class="thirdtext">LET’S CONNECT</h3>
...
+              src="./images/twitter.png"
+              alt="twitter"
+              class="Twitterlogo2"
+            />
+
+            <img src="./images/Mail.png" alt="Mail" class="Maillogo2" />
+          </div>
+
+          <button class="lasttxt2">get my resume</button>
+        </div>
```
Comment (elmar8287) [inline Portfolio.html]: - [ ] Please consider this button as others, required by project style. You can see, that your version is different from the original one:
> Your version
Screen Shot 2022-05-30 at 17 46 05

> Original one
Screen Shot 2022-05-30 at 17 49 53

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

I want to highlight:

- no linter issues

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Portfolio.html [hunk 6]
```diff
@@ -44,6 +44,219 @@ <h3 class="thirdtext">LET’S CONNECT</h3>
...
+            <img
+              src="./images/twitter.png"
+              alt="twitter"
+              class="Twitterlogo2"
+            />
+
+            <img src="./images/Mail.png" alt="Mail" class="Maillogo2" />
+          </div>
+
+          <button class="lasttxt2">get my resume</button>
```
Comment (SajjadAhmad14) [inline Portfolio.html]: Please make sure that this button is exactly the same as in the design. Text is not centered both horizontally and vertically. The button does not have a hover and active effect as in the design.

Yours:
your-btn

Design:

resume

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Michael,
### Great work on making the changes requested by a previous reviewer 👏 
### You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Therealadetunji , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**